server/views/referrals/checkAnswers.njk

```diff
@@ -33,69 +33,68 @@
 
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
-      <p class="govuk-body">Read through this page and check that the information provided for each answer is accurate. When you feel the application is complete, submit it.</p>
+    <p class="govuk-body">Read through this page and check that the information provided for each answer is accurate. When you feel the application is complete, submit it.</p>
```

Comment:
I think most of the indent changes in this file/commit aren't wanted?

I added a setting to hopefully automate the indents in VSCode, but I'm not sure if you use VSCode?